Tasks/Common/webdeployment-common/msdeploy.exe.config

```diff
@@ -0,0 +1,6 @@
+<configuration>
+  <startup  useLegacyV2RuntimeActivationPolicy="true" >
+    <supportedRuntime version="v4.0" sku=".NETFramework,Version=v4.0" />
+    <supportedRuntime version="v2.0.50727" />
```

Comment:
Why do we need to 2.0?

Reply:
We are following the same configuration that we have for installed MSDEPLOY.